core/pta/rtc.c

```diff
@@ -27,6 +27,18 @@ static void rtc_pta_copy_time_from_optee(struct pta_rtc_time *pta_time,
 	pta_time->tm_wday = optee_time->tm_wday;
 }
 
+static void rtc_pta_copy_time_to_optee(struct optee_rtc_time *optee_time,
```

Comment:
Could you move this change to commit *"core: pta: add alarm-related operations to RTC PTA"*?

I think commit *"core: pta: verify RTC PTA data buffer is aligned with struct pta_rtc_time"* should only deal with alignment verification and have a `Fixes: cea1eb0bc90e ("pta: add PTA for RTC")` tag.